mmdet/utils/misc.py

```diff
@@ -1,8 +1,12 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import glob
+import os
 import os.path as osp
 import warnings
 
+import mmcv
```

Comment:
Please write docs about this new feature [here](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/3_exist_data_new_model.md#prepare-the-standard-dataset)